tt_eager/tt_dnn/op_library/conv/kernels/bmm_tilize_untilize_all_weights_in_l1_single_output_block_width_dim.cpp

```diff
@@ -155,10 +155,7 @@ void MAIN {
         uint32_t in1_index_inner_dim_h_offset = 0;
 
         #ifdef PACK_RELU
-        if constexpr ((tilize_in0 || spill) && !untilize_out) {
-            // for each output block we start we relu disabled so that intermediate results are not relu'd
-            PACK(( llk_pack_relu_config(ReluType::NO_RELU) ));
-        }
+        PACK(( llk_pack_relu_config(ReluType::NO_RELU) ));
```

Comment:
where was relu being applied that it was not supposed to?

Reply:
if we don’t have spill and reload and bias is enabled we don’t disable relu for the next block